m/reading_tools/dbacat.m

```diff
@@ -193,7 +193,7 @@
   meta.units   = all_units(sensors_idx);
   meta.bytes   = all_bytes(sensors_idx);
   
-  
+
```

Comment:
White space unintentional change here?
`git reset v1.1.1 m/reading_tools/dbacat.m` to revert it.